src/test/java/org/gridsuite/loadflow/server/LoadFlowControllerTest.java

```diff
@@ -280,6 +280,37 @@ public void testGetLimitViolations() throws Exception {
             });
             assertLimitViolationsEquals(LimitViolationsMock.limitViolations, limitViolations, network);
         }
+
+        // get loadflow limit types
+        MvcResult mvcResult = mockMvc.perform(get("/" + VERSION + "/results/{resultUuid}/limit-types", RESULT_UUID))
+                .andExpectAll(
+                        status().isOk(),
+                        content().contentType(MediaType.APPLICATION_JSON)
+                ).andReturn();
+        List<LimitViolationType> limitTypes = mapper.readValue(mvcResult.getResponse().getContentAsString(), new TypeReference<>() { });
+        assertEquals(1, limitTypes.size());
+        assertTrue(limitTypes.contains(LimitViolationType.CURRENT));
+
+        // get loadflow branch sides
+        mvcResult = mockMvc.perform(get("/" + VERSION + "/results/{resultUuid}/branch-sides", RESULT_UUID))
+                .andExpectAll(
+                        status().isOk(),
+                        content().contentType(MediaType.APPLICATION_JSON)
+                ).andReturn();
+        List<TwoSides> sides = mapper.readValue(mvcResult.getResponse().getContentAsString(), new TypeReference<>() { });
+        assertEquals(2, sides.size());
+        assertTrue(sides.contains(TwoSides.ONE));
+        assertTrue(sides.contains(TwoSides.TWO));
+
+        // get loadflow computing status
+        mvcResult = mockMvc.perform(get("/" + VERSION + "/results/{resultUuid}/computation-status", RESULT_UUID))
+                .andExpectAll(
+                        status().isOk(),
+                        content().contentType(MediaType.APPLICATION_JSON)
+                ).andReturn();
+        List<LoadFlowResult.ComponentResult.Status> status = mapper.readValue(mvcResult.getResponse().getContentAsString(), new TypeReference<>() { });
+        assertEquals(1, status.size());
+        assertTrue(status.contains(LoadFlowResult.ComponentResult.Status.CONVERGED));
```

Comment:
I don't think this should be put into a testGetLimitViolations function : this has nothing to do with limit violations.